--- FAIL: TestWatch (0.00s)
panic: Trace folder rewritten_trace_6 does not exist. [recovered]
	panic: Trace folder rewritten_trace_6 does not exist.

goroutine 158 [running]:
testing.tRunner.func1.2({0x1fb7b80, 0xc00037aa30})
	/home/plantontable/Mari/ADVOCATE/go-patch/src/testing/testing.go:1631 +0x24a
testing.tRunner.func1()
	/home/plantontable/Mari/ADVOCATE/go-patch/src/testing/testing.go:1634 +0x45d
panic({0x1fb7b80?, 0xc00037aa30?})
	/home/plantontable/Mari/ADVOCATE/go-patch/src/runtime/panic.go:770 +0x132
advocate.EnableReplay(0x6, 0xc?)
	/home/plantontable/Mari/ADVOCATE/go-patch/src/advocate/advocate.go:297 +0x3a9
k8s.io/kubernetes/pkg/registry/resource/resourceclaimparameters/storage.TestWatch(0xc000702fc0)
	/home/plantontable/Mari/kubernetes/pkg/registry/resource/resourceclaimparameters/storage/storage_test.go:126 +0x4e
testing.tRunner(0xc000702fc0, 0x2523190)
	/home/plantontable/Mari/ADVOCATE/go-patch/src/testing/testing.go:1689 +0xfb
created by testing.(*T).Run in goroutine 1
	/home/plantontable/Mari/ADVOCATE/go-patch/src/testing/testing.go:1742 +0x40d
FAIL	k8s.io/kubernetes/pkg/registry/resource/resourceclaimparameters/storage	0.107s
FAIL
